docker-machine-nfs.sh

```diff
@@ -91,21 +91,21 @@ EOF
 # @args:    error-message
 echoError ()
 {
-  echo "\033[0;31mFAIL\n\n$1 \033[0m"
+  printf "\033[0;31mFAIL\n\n$1 \033[0m\n"
```

Comment:
Those changes (and other similar changes) are not really related to the support for HaneWIN ... I prefer to keep the PRs related to one topic and not mix things. Another PR could be created with only semantic changes .

I'd also be interested why this change would be required ... ?!

Reply:
At some places you're using `printf` and at some `echo`. 

Firstly I wanted to unify it, and secondly on WSL and on Linux bash `echo` doesn't print symbols with backslash correctly. Instead of plain `echo` you have to use either `echo -e` or `printf`.

Example:
```
jerry@JerryPC /c/Users/Jerry/projects/docker-machine-nfs (support-for-wsl) $ echo "\033[0;31mTest\n\n$1 \033[0m"                                                                                                                
\033[0;31mTest\n\n \033[0m 
jerry@JerryPC /c/Users/Jerry/projects/docker-machine-nfs (support-for-wsl) $ echo -e "\033[0;31mTest\n\n$1 \033[0m"                                                                                                             
Test 
     
     
jerry@JerryPC /c/Users/Jerry/projects/docker-machine-nfs (support-for-wsl) $ printf "\033[0;31mTest\n\n$1 \033[0m\n"
Test 
     
     
jerry@JerryPC /c/Users/Jerry/projects/docker-machine-nfs (support-for-wsl) $  
```

I'm taking those changes back and will include in another PR.